lib/python/pyflyby/_cmdline.py

```diff
@@ -37,16 +39,10 @@ def _sigpipe_handler(*args):
     raise SystemExit(1)
 
 
-def parse_args(
-    addopts=None, import_format_params=False, modify_action_params=False, defaults=None
-):
+def parse_args(addopts=None, import_format_params=False, modify_action_params=False):
```

Comment:
I think this is fine to remove because it is in a private `_cmdline` module. Alternatively the `defaults` could be kept for now but raise a warning if passed.

Reply:
> I think this is fine to remove because it is in a private _cmdline module
Exactly what I was thinking too :+1: